src/main/java/com/endava/soa_5ed/task1_basics/Ex2.java

```diff
@@ -0,0 +1,18 @@
+package com.endava.soa_5ed.task1_basics;
+
+import java.util.Scanner;
+
+public class Ex2 {
```

Comment:
Same as above, you could find a more appropriate class name to be more representable for the code written in it

Reply:
Done.